hc/settings.py

```diff
@@ -1,24 +1,24 @@
 """
 Django settings for hc project.
-
 Generated by 'django-admin startproject' using Django 1.8.2.
-
 For more information on this file, see
 https://docs.djangoproject.com/en/1.8/topics/settings/
-
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/1.8/ref/settings/
 """
 
+import dj_database_url
 import os
 import warnings
+import sendgrid
+
 
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
 HOST = "localhost"
 SECRET_KEY = "---"
 DEBUG = True
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['hc-codedragons.herokuapp.com']
```

Comment:
Hi Abiodun, the project environment settings are defined in DJANGO_SETTINGS_MODULE in manage.py in the root folder. It is defined as follows: os.environ.setdefault("DJANGO_SETTINGS_MODULE", "hc.settings")
This points to the settings.py in the folder hc which is why we put those settings there.